src/core/Akka/Actor/ChildrenContainer/Internal/ChildrenContainerBase.cs

```diff
@@ -188,19 +187,18 @@ public bool Contains(IActorRef actor)
         /// <param name="sb">TBD</param>
         /// <param name="kvp">TBD</param>
         /// <param name="index">TBD</param>
-        protected void ChildStatsAppender(StringBuilder sb, KeyValuePair<string, IChildStats> kvp, int index)
+        protected static void ChildStatsAppender(StringBuilder sb, KeyValuePair<string, IChildStats> kvp, int index)
```

Comment:
Not a breaking change - moving to `static` would only change inheritors, not callers.

Reply:
(and there could be no inheritors of this method before.)